src/server/api/routes/users-router.js

```diff
@@ -122,4 +122,54 @@ router.post('/', (req, res) => {
     });
 });
 
+/**
+ * @swagger
+ * /users/{ID}:
+ *  patch:
+ *    summary: edit a user
+ *    description:
+ *      Will edit a user.
+ *    produces: application/json
+ *    parameters:
+ *      - in: path
+ *        name: ID
+ *        description: ID of the user to patch.
+ *      - in: body
+ *        name: user
+ *        description: The user to edit.
+ *        schema:
+ *          type: object
+ *          properties:
+ *            profileImage:
+ *              type: string
+ *            userName:
+ *              type: string
+ *            email:
+ *              type: string
+ *            lastSeen:
+ *              type: string
```

Comment:
good job :), one minor thing I think its not a good idea that we allow anyone to update lastSeen, and updatedAt at these 2 fields are kinda logic dependent you can't just allow anyone to change them